packages/table/src/cell/formats/jsonFormat.tsx

```diff
@@ -29,9 +28,9 @@ export interface IJSONFormatProps extends ITruncatedFormatProps {
     stringify?: (obj: any) => string;
 }
 
-@PureRender
-export class JSONFormat extends React.Component<IJSONFormatProps, {}> {
+export class JSONFormat extends React.PureComponent<IJSONFormatProps, {}> {
```

Comment:
now that this is a thing, we should replace the `PureRender` annotation in other places in BP. for another PR!